javascript/javascript1/week3/main.js

```diff
@@ -0,0 +1,274 @@
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+for (var i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+    i--;
```

Comment:
i-- is needed so that once the element A is removed by splice and the next element B takes A's position then if we do not decrease i then we will skip B.